applications/nvidia_video_codec/nvc_decode/README.md

```diff
@@ -52,10 +52,7 @@ ffmpeg -i surgical_video.mp4 -c:v copy -f h264 surgical_video.264
 ```
 
 ### Verification
-Use the provided test script to verify your H.264 file:
-```bash
-python3 test_h264_reader.py
-```
+You can verify your H.264 file format by running the application with a small test file first.
```

Comment:
Is `test_h264_reader.py` no longer supported? Or, could we provide more concrete verification instructions?